asm/Kptfile

```diff
@@ -1,123 +1,221 @@
 apiVersion: kpt.dev/v1alpha1
 kind: Kptfile
 metadata:
-  name: asm
+  name: asm-istiod-15
+packageMetadata:
+  shortDescription: ASM Networking 1.5
 openAPI:
   definitions:
-    io.k8s.cli.setters.gcloud.compute.location:
-      type: string
+    io.k8s.cli.setters.meshca-address:
```

Comment:
use `anthos.servicemesh.meshca-address` for the setter instead? Setters will be user-facing APIs, better to keep consistent with all servicemesh configs.